### PR TITLE
"micro:bit 2024 Update" blog post

### LINKED ISSUE #10125: Blog Post - Micro:bit 2024 update
Hi Galen, can you please prep this blog post?  Would like to go out next Tuesday.  Files are all here:  https://1drv.ms/f/s!AmMIW5Hxi0Rtgac1yTqJ9Q7XABQJCQ?e=eo8Zk7

There may be some changes/edits coming back from the micro:bit Foundation...

Thanks!
Jaq.

### PR DESCRIPTION
**Pull request by ganicke**
Import, format, adjust images, edit, and otherwise prepare the "micro:bit 2024 Update" blog post.

Closes #10125

### CODE REVIEW

File: docs/blog/microbit/2024-update.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# MakeCode micro:bit 2024 Update
```
Comment (ganicke) [inline docs/blog/microbit/2024-update.md:1]: "MakeCode **for** microbit"?

File: docs/blog/microbit/2024-update.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Version History
+
+Have you ever been working on a project when all of a sudden, your evil twin takes over and completely destroys your work and introduces a ton of bugs? Well, never fear, Version History is here! You can access previous versions of your code through the Project Settings menu in the upper right corner.
+
+![Project settings menu](/static/blog/microbit/2024-update/project-settings-menu.jpg)
+
+In the **Version History** pane, you can see the timestamps of when your code was automatically saved. You can click on the different times to see the version of your code at that time, and you can see when you shared your project. Using the buttons at the top, you can save a copy of any version of your program, or you can restore an older version of your code (before your evil twin took over).
+
+&lt;**VersionHistory.gif - will be updated**&gt;
```
Comment (ganicke) [inline docs/blog/microbit/2024-update.md:27]: Image pending update...

File: docs/blog/microbit/2024-update.md (hunk 1 above)
Comment (Jaqster) [reply docs/blog/microbit/2024-update.md:1]: Ah yes!  Good catch.  Should be "MakeCode for the micro:bit"

**Comment by Jaqster** [comment]
Also, can you add a section on WebUSB?

**WebUSB reliability**
We released the new “one-click-to-download” functionality supported by WebUSB [last year](https://makecode.com/blog/microbit/2023-release).  Since then, we’ve done a lot of work to ensure that this method of pairing the micro:bit to the browser is reliable and just works all the time across different platforms.  Please try it out – it should be very stable.  If you are still experiencing any issues with downloading programs to the micro:bit through this method, please let us know by filing a ticket at the [micro:bit support desk](https://support.microbit.org/support/tickets/new). 
 <Download dialog.jpg>

File: docs/blog/microbit/2024-update.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
```
Comment (Jaqster) [inline docs/blog/microbit/2024-update.md:28]: Think we're missing a ##Title here...

File: docs/blog/microbit/2024-update.md (hunk 2 above)
Comment (ganicke) [reply docs/blog/microbit/2024-update.md]: Added updated `version-history gif`.

**Review by Jaqster — APPROVED** [review]
Looks great!  Thank you Galen!

**Pull request merged**